### ISSUE TITLE
memory leak detected

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/forces-and-motion-basics#277]
Starting yesterday, CT started to detect a consistent memory leak
```
forces-and-motion-basics : phet-io-fuzz : require.js : run
Query: brand=phet-io&phetioStandalone&ea&fuzz&memoryLimit=1000
Uncaught Error: Assertion failed
Error: Assertion failed
    at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/snapshot-1548200844544/assert/js/assert.js:22:13)
    at updateAppliedForcePosition (https://bayes.colorado.edu/continuous-testing/snapshot-1548200844544/forces-and-motion-basics/js/motion/view/PusherNode.js?bust=1548202228262:206:17)
    at Array.<anonymous> (https://bayes.colorado.edu/continuous-testing/snapshot-1548200844544/forces-and-motion-basics/js/motion/view/PusherNode.js?bust=1548202228262:282:9)
    at Emitter.emit3 (https://bayes.colorado.edu/continuous-testing/snapshot-1548200844544/axon/js/Emitter.js?bust=1548202228262:303:54)
    at NumberProperty._notifyListeners (https://bayes.colorado.edu/continuous-testing/snapshot-1548200844544/axon/js/Property.js?bust=1548202228262:239:27)
    at NumberProperty.setValueAndNotifyListeners (https://bayes.colorado.edu/continuous-testing/snapshot-1548200844544/axon/js/Property.js?bust=1548202228262:219:12)
    at NumberProperty.set (https://bayes.colorado.edu/continuous-testing/snapshot-1548200844544/axon/js/Property.js?bust=1548202228262:152:14)
    at NumberProperty.set (https://bayes.colorado.edu/continuous-testing/snapshot-1548200844544/axon/js/NumberProperty.js?bust=1548202228262:84:13)
    at NumberProperty.set value [as value] (https://bayes.colorado.edu/continuous-testing/snapshot-1548200844544/axon/js/Property.js?bust=1548202228262:308:34)
    at Emitter.<anonymous> (https://bayes.colorado.edu/continuous-testing/snapshot-1548200844544/scenery-phet/js/FineCoarseSpinner.js?bust=1548202228262:62:29)
id: Bayes Chrome
```

Reports started at 6:47 PM 1/22/19.

**Comment by jessegreenberg** [comment]
Taken over about 3 minutes of fuzz testing:
![image](https://user-images.githubusercontent.com/6396244/51622736-3c641400-1f05-11e9-8eb2-f3d66c9a0b34.png)


I am not seeing much of a leak, but these heap sizes are much larger than I usually see.

**Comment by jessegreenberg** [comment]
Continuing to run, heap size is up to 92 MB.

**Comment by jessegreenberg** [comment]
Doing other things and refreshed the sim in another window. Now heap sizes are down by about half. In the original test I see two entries called "WeakMap" and "Detached Window" that are taking up a large amount of memory.
![image](https://user-images.githubusercontent.com/6396244/51624547-117bbf00-1f09-11e9-8cf9-7244625dbde6.png)

These don't exist in my later tests. What are these?

**Comment by jessegreenberg** [comment]
I refreshed and these things came back, with a much larger heap size.

**Comment by jessegreenberg** [comment]
Now I can't get these to show up at all. Very weird. And, I realized that I missread the error

> Uncaught Error: Assertion failed

Caught during memory test fuzzing, but error has nothing to do with memory. I think this has been fixed. And since I am not seeing the WeakMap or the large memory usage anymore locally I don't know what to do about that. If this error disapears on CT ill just close.
